### PR TITLE
Translation updates

### PR DESCRIPTION
**Pull request by ManlyMarco**

### CODE REVIEW

File: Translation/Clean Intro.txt [hunk 1]
```diff
@@ -71,43 +71,43 @@
...
+も、もう……じろじろ見てないでなんとか言ってよ……=G-Geez...\nStop staring at me and say something already...
 良いんだよね？=\nCan I?
 
 		# if "*nod*" => Continue story
 
 		# Continuation
 特に断る理由もないので頷くと、そそくさと浴室へ向かっていく。=Since there was no reason to refuse, you nodded and let her go to the bathroom.
 脱ぎ散らかされた制服のポケットからは、彼女のパスケースがこぼれ落ちていた。=Her pass case fell down from the pocket of the uncluttered uniform.
-元に戻しておこうと近付くと、表面に収められていた学生証が目に留まる。=While you were putting it back, you couldn't help but see her Student ID that was showing on the cover.
+元に戻しておこうと近付くと、表面に収められていた学生証が目に留まる。=While you were putting it back, you couldn't help but saw her Student ID that was showing on the cover.
```
Comment (WataThaBradicus) [inline Translation/Clean Intro.txt:82]: I think "see" sounds better here even if we did decide on past tense for everything.

File: Translation/Clean Intro.txt [hunk 2]
```diff
@@ -71,43 +71,43 @@
...
 
 		#If "caress her"
 ん……。=Hmm...
-そっと頭を撫でると、夕は穏やかな表情を浮かべて、静かな寝息を立て始めた。=You gently caressed her head, her expression eases and she returns to sleep shortly.
-安らかな寝顔を眺めている内に、あなたにもだんだんと眠気の波が押し寄せてくる。=As you watch her peaceful sleeping face, waves of drowsiness gradually washed over.
+そっと頭を撫でると、夕は穏やかな表情を浮かべて、静かな寝息を立て始めた。=You gently caressed her head, her expression eased and she returned to sleep shortly.
+安らかな寝顔を眺めている内に、あなたにもだんだんと眠気の波が押し寄せてくる。=As you watched her peaceful sleeping face, waves of drowsiness gradually washed over.
 その夜。あなたは夕と寄り添いながら、絶えず心に付きまとっていた不安も忘れて、=That night, while sleeping with her, you forgot the worries that were constantly on your mind.
-本当に久しぶりに、深い眠りについた。=You had a slept well for the first time in forever.
+本当に久しぶりに、深い眠りについた。=You had a well sleep for the first time in forever.
```
Comment (WataThaBradicus) [inline Translation/Clean Intro.txt:104]: "You slept well for the first time in forever."

I think this is better?

Comment (ManlyMarco) [reply Translation/Clean Intro.txt:104]: It should either be "You slept well for the first time in forever." or "You had some good sleep for the first time in forever."

File: Translation/Clean Intro.txt (hunk 1 above)
Comment (ManlyMarco) [reply Translation/Clean Intro.txt:82]: see is correct here.

**Pull request merged**